laser-reflection.js

```diff
@@ -0,0 +1 @@
+// reflect the laser pointer pls
```

Comment:
great!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!1

Reply:
Thx for the feedback